packages/@webex/plugin-cc/src/services/core/Utils.ts

```diff
@@ -0,0 +1,20 @@
+import * as Err from './Err';
+import {WebexRequestPayload} from '../../types';
+import {WCC_API_GATEWAY} from '../constants';
+
+const getCommonErrorDetails = (errObj: WebexRequestPayload) => {
+  return {
+    trackingId: errObj?.headers?.trackingid || errObj?.headers?.TrackingID,
+    msg: errObj?.body,
+  };
+};
+
+export const createErrDetailsObject = (errObj: WebexRequestPayload) => {
+  const details = getCommonErrorDetails(errObj);
+
+  return new Err.Details('Service.reqs.generic.failure', details);
+};
+
+export const getRoutingHost = () => {
```

Comment:
We can directly import the constant instead of using this function

Reply:
addressed